src/Signal/Extra.elm

```diff
@@ -34,6 +35,9 @@ module Signal.Extra
 # Flipped fancy map
 @docs (~>)
 
+# Mapping
+@docs andMap
+
```

Comment:
Can you add the `(~>)` to this section and remove the **Flipped fancy map** section?